docs/content/install.md

```diff
@@ -69,6 +70,53 @@ You will need to create a [PowerShell Profile][ps-link] if you do not have one.
 iwr "https://cdn.porter.sh/canary/install-windows.ps1" -UseBasicParsing | iex
 ```
 
+# v1 Prerelease
+
+We would love for you to try out v1 prelease and send us any feedback that you have!
+Keep in mind that the v1 prerelease is not suitable for running with production workloads, and that data migrations will not be provided or supported for v1 prerelease.
+The prerelease is intended for you to try out the new features in Porter and provide feedback. It won't work with existing installations.
```

Comment:
Prereleases are intended for you to try out potential new features in Porter and provide feedback about the direction of the feature. They won't work with existing installations.